vuepress/docs/.vuepress/next.js

```diff
@@ -3,67 +3,115 @@ module.exports = {
         return [
             {
                 title: 'Introduction',
-                path: path,
+                children: [
+                    {
+                        title: 'Getting Started',
+                        path: path + 'getting-started/',
+                    },
+                    {
+                        title: 'Entando Architecture',
```

Comment:
I played with this a bit and I think we're really close. The only thing that's bugging me is that the Entando Architecture page is the default and it's second in the list. Thinking it through, that is the current default page if you just come straight to /docs. I think that still makes sense but the name isn't quite right then. I'm thinking we should make it first in the list, rename it to Overview.  Then we have Overview, Getting Started, Concepts in the list.